app/config/locale/translations/pl.json

```diff
@@ -3,30 +3,30 @@
     "settings.locale": "pl",
     "settings.direction": "ltr",
     "emails.sender": "Zespół %s",
-    "emails.verification.subject": "",
-    "emails.verification.hello": "",
-    "emails.verification.body": "",
-    "emails.verification.footer": "",
-    "emails.verification.thanks": "",
-    "emails.verification.signature": "",
-    "emails.magicSession.subject": "",
-    "emails.magicSession.hello": "",
-    "emails.magicSession.body": "",
-    "emails.magicSession.footer": "",
-    "emails.magicSession.thanks": "",
-    "emails.magicSession.signature": "",
-    "emails.recovery.subject": "",
-    "emails.recovery.hello": "",
-    "emails.recovery.body": "",
-    "emails.recovery.footer": "",
-    "emails.recovery.thanks": "",
-    "emails.recovery.signature": "",
-    "emails.invitation.subject": "",
-    "emails.invitation.hello": "",
-    "emails.invitation.body": "",
-    "emails.invitation.footer": "",
-    "emails.invitation.thanks": "",
-    "emails.invitation.signature": "",
+    "emails.verification.subject": "Weryfikacja konta",
+    "emails.verification.hello": "Cześć {{name}}",
+    "emails.verification.body": "Przejdź do tego linku, aby zweryfikować swój adres e-mail.",
+    "emails.verification.footer": "Jeśli to nie Ty prosiłeś o zweryfikowanie tego adresu, możesz zignorować tę wiadomość.",
+    "emails.verification.thanks": "Dziękujemy",
+    "emails.verification.signature": "Zespół {{project}}",
+    "emails.magicSession.subject": "Logowanie",
+    "emails.magicSession.hello": "Cześć,",
+    "emails.magicSession.body": "Przejdź do tego linku, aby zalogować się.",
+    "emails.magicSession.footer": "Jeśli to nie Ty prosiłeś o logowanie przy użyciu tego adresu e-mail, możesz zignorować tę wiadomość.",
+    "emails.magicSession.thanks": "Dziękujemy",
+    "emails.magicSession.signature": "Zespół {{project}}",
+    "emails.recovery.subject": "Resetowanie hasła",
+    "emails.recovery.hello": "Cześć {{name}}",
+    "emails.recovery.body": "Przejdź do tego linku, aby zresetować hasło dla {{project}}.",
+    "emails.recovery.footer": "Jeśli to nie Ty prosiłeś o zresetowanie swojego hasła, możesz zignorować tę wiadomość.",
+    "emails.recovery.thanks": "Dziękujemy",
+    "emails.recovery.signature": "Zespół {{project}}",
+    "emails.invitation.subject": "Zaproszenie do zespołu %s w %s",
+    "emails.invitation.hello": "Cześć",
```

Comment:
```suggestion
    "emails.invitation.hello": "Cześć,",
```

Reply:
I'm not sure if the comma should be at the end, because in English translation there is no comma at the end. All punctuation is based on the English translation file. 
https://github.com/appwrite/appwrite/blob/master/app/config/locale/translations/en.json#L25